### PR TITLE
Updated plot to ggplot

### PR DESCRIPTION
**Pull request by s1nzy**
added libray
changed plot description
switch lattice to ggplot

### CODE REVIEW

File: Initial_script.R [hunk 1]
```diff
@@ -5,18 +5,14 @@
...
-
-xyplot(mpg ~ hp, 
-       data = mtcars,
-       col = "purple")
-
+# description: miles per gallon by horse power:
 
+ggplot(data = mydata)+
+  geom_point(aes(x = hp, y = mpg), col = "purple")+
+  theme_bw()
```
Comment (fake-sinzi) [inline Initial_script.R]: Amazing work! Add the labels for the axes please before merging.

File: Initial_script.R [hunk 2]
```diff
@@ -5,18 +5,14 @@
 # description: cars plots with ggplot
 # ---
 
-# library(lattice)
```
Comment (fake-sinzi) [inline Initial_script.R]: Remove this, as it is no longer needed

Comment (s1nzy) [reply Initial_script.R]: done!

File: Initial_script.R (hunk 1 above)
Comment (s1nzy) [reply Initial_script.R]: You are right. This was added now

**Pull request merged**